### PR TITLE
compatiblity for artix :D

### PR DESCRIPTION
**Pull request by Damaj301damaj-lol**
Artix is different from arch, especially in stuff like keys and stuff, this adds safety stuff for artix users

### CODE REVIEW

File: install-base.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ if [[ -f "/usr/bin/emerge" ]]; then
 elif [[ -f "/usr/bin/apt" ]]; then
 	echo "Found distro: Debian" && distro=debian
 elif [[ -f "/usr/bin/pacman" ]]; then
-	echo "Found distro: Arch" && distro=arch
+	echo "Found distro: Arch or Artix" && distro=arch
```
Comment (jornmann) [inline install-base.sh:36]: This could be changed to "Arch based"

File: install-base.sh [hunk 2]
```diff
@@ -61,9 +61,16 @@ fi
 
 # Arch
 if [[ -f "/usr/bin/pacman" ]]; then
+	if [[ -f "/etc/rc.conf" ]]; then
```
Comment (jornmann) [inline install-base.sh:64]: Couldn't you just check /etc/os-release for a more accurate check?

File: install-base.sh [hunk 3]
```diff
@@ -61,9 +61,16 @@ fi
...
+		echo "Found Artix"
+		pacman -Sy || exit 1
+		pacman -S libxinerama xorg-server terminus-font ttf-font-awesome base-devel picom alsa-utils firefox maim git xclip feh neovim xorg-server xorg-xinit newsboat htop zsh yt-dlp || exit && echo "Installed dependencies"
+	else
+	echo"Found Arch"
```
Comment (jornmann) [inline install-base.sh:69]: space

**Review by jornmann — CHANGES_REQUESTED** [review]
The Artix check might not be very accurate, so changes should be made

File: install-base.sh (hunk 3 above)
Comment (Damaj301damaj-lol) [reply install-base.sh:69]: fine you get that

File: install-base.sh (hunk 2 above)
Comment (Damaj301damaj-lol) [reply install-base.sh:64]: no, if os-release would be used, just it for every distro bruh

**Comment by jornmann** [comment]
???? I can't understand you

File: install-base.sh (hunk 1 above)
Comment (Damaj301damaj-lol) [reply install-base.sh:36]: make sence

File: install-base.sh (hunk 2 above)
Comment (jornmann) [reply install-base.sh:64]: I can't understand you

**Comment by Damaj301damaj-lol** [comment]
redoing the work upon jorns request

**Pull request closed without merging**